stereo_camera:
  baseline: 0.1
  resolution:
    width: 640
    height: 480
  focal_length: 15
  focus_distance: 0.6
  f_stop: 240
  clipping_range:
    near: 0.1
    far: 100000

rtx_lidar:
  config_file: "OS1_REV7_128ch10hz2048res"



